lib/types.ts

```diff
@@ -19,6 +19,7 @@ export interface Contributor {
   intern: boolean;
   operations: boolean;
   courses_completed: string[];
+  first_activity: Date | string;
```

Comment:
Stick with any one of the type either Date or string since this is a computed value.